includes/Commission/OrderCommission.php

```
@@ -116,27 +116,27 @@
-		}
+        if ( $this->get_should_adjust_refund() ) {
+            $this->set_should_adjust_refund( false );
+            $this->calculate();
+            $should_reset_adjust_refund = true;
+        }
 
         $this->ensure_commissions_are_calculated();
 
         $refund_commission = new Commission();
 
         foreach ( $refund->get_items() as $item_id => $refund_item ) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$item_id'. (Unused Code Rules)

(UnusedLocalVariable)